internal/compiler-bridge-test/src/test/scala/sbt/internal/inc/ExtractUsedNamesPerformanceSpecification.scala

```diff
@@ -27,6 +27,7 @@ class ExtractUsedNamesPerformanceSpecification extends UnitSpec {
   val scalaDiff = Set("Any", "Nothing", "_root_", "StringAdd", "Option")
 
   it should "be executed in reasonable time" in {
+    pending
```

Comment:
Shall we open tickets for this and the next test too?

Reply:
I think this was failing on me locally, but I reenabled it and it seems to be doing ok.